### ISSUE TITLE
Tests cause segmentation faults

### ISSUE THREAD

**Issue by olebole** [spacetelescope/drizzle#7]
When running `python -c "import drizzle; exit(drizzle.test())"`, I get a segmentation fault. With gdb:
```
drizzle/tests/test_cdrizzle.py 
Program received signal SIGSEGV, Segmentation fault.
test_cdrizzle (self=<optimized out>, args=<optimized out>)
    at drizzle/src/cdrizzleapi.c:364
364	  dat = (PyArrayObject *)PyArray_ContiguousFromAny(data, PyArray_FLOAT, 2, 2);
```
When I disable this test, I get another one:
```
drizzle/tests/test_drizzle.py 
Program received signal SIGSEGV, Segmentation fault.
tdriz (obj=<optimized out>, args=0x7fffea0fcb48, keywords=0x7fffea134910)
    at drizzle/src/cdrizzleapi.c:95
95	  img = (PyArrayObject *)PyArray_ContiguousFromAny(oimg, PyArray_FLOAT, 2, 2);
```
This happens for both Python 2.7 and Python 3.5.

**Comment by bernie-simon** [comment]
I haven't been able to duplicate your errors. There must be something different about the configuration on your site and mine. Could you say a bit more about where you are running drizzle? Usually I run the drizzle tests from the directory it is installed in with the command 

python setup.py test

but your command should work as well.

**Comment by olebole** [comment]
Ah, OK. The reason is that I use the released version 1.1, while you tested the current version. The relevant change is 656dcad7,  probably the `import_array()` at the end. When I apply this patch, the tests don't crash anymore and all pass. So, I will add this patch to the Debian package.
I would however recommend to do a new release that includes this patch.

**Comment by bernie-simon** [comment]
I've created a new version of the drizzle package, 1.3, with all the changes to date.
